src/browser/index.js

```diff
@@ -39,6 +40,9 @@ if (isReload) {
   window.application.address(UI.LiveReload);
 }
 
+document.body.classList.toggle('use-native-titlebar',
+                               Runtime.useNativeTitlebar());
+
```

Comment:
I'd prefer to keep the logic that tells if native titlebar is used or not, isolated.

Eventually, the runtime will tell us what kind of titlebar is used, and we will just need to change what's within runtime.js.